config/Run3_2022EE/samples.yaml

```diff
@@ -44,3 +38,990 @@ JetMET_Run2022F:
+DYto2Tau_MLL-6000_powheg:
+  sampleType: DY
+  crossSectionStitch: DYto2Tau_MLL-6000_powheg
+  generator: powheg
+  miniAOD: /DYto2Tau_MLL-6000_TuneCP5_13p6TeV_powheg-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+DYto2TautoMuTauh_M-50_madgraphMLM:
+  sampleType: DY
+  crossSectionStitch: DYto2TautoMuTauh_M-50_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /DYto2TautoMuTauh_M-50_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v3/MINIAODSIM
+
+############## H ##############
+GluGluHToTauTau_M125_amcatnloFXFX:
+  sampleType: H
+  crossSection: GluGluHToTauTau_M125
+  generator: amcatnloFXFX
+  miniAOD: /GluGluHto2Tau_M-125_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+GluGluHToWWTo2L2Nu_M125:
+  sampleType: H
+  crossSection: GluGluHToWWTo2L2Nu_M125
+  generator: powheg
+  miniAOD: /GluGluHto2Wto2L2Nu_M-125_TuneCP5_13p6TeV_powheg-jhugen752-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+
+VBFHToWWTo2L2Nu_M125:
+  sampleType: H
+  crossSection: VBFHToWWTo2L2Nu_M125
+  generator: powheg
+  miniAOD: /VBFHto2Wto2L2Nu_M-125_TuneCP5_13p6TeV_powheg-jhugen752-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+
+############## QCD ##############
+QCD_HT70to100_MLM:
+  sampleType: QCD
+  crossSection: QCD_HT70to100_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-70to100_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+QCD_HT100to200_MLM:
+  sampleType: QCD
+  crossSection: QCD_HT100to200_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-100to200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+QCD_HT200to400_MLM:
+  sampleType: QCD
+  crossSection: QCD_HT200to400_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-200to400_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+QCD_HT400to600_MLM:
+  sampleType: QCD
+  crossSection: QCD_HT400to600_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-400to600_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+QCD_HT600to800_MLM:
+  sampleType: QCD
+  crossSection: QCD_HT600to800_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-600to800_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+QCD_HT800to1000_MLM:
+  sampleType: QCD
+  crossSection: QCD_HT800to1000_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22EEMiniAODv4-130X_mcRun3_2022_realistic_postEE_v6-v2/MINIAODSIM
+QCD_HT1000To1200_MLM:
+  sampleType: QCD
+  crossSection: QCD_HT1000To1200_MLM
```

Comment:
`QCD_HT1000To1200_MLM` is not defined

Reply:
What do you mean not defined? Its in the crossSections yaml (line 270 in my local version)

Reply:
You're so smart, keys are case sensitive, fixed to lowercase 'To'